frontend/src/scenes/instance/AsyncMigrations/index.tsx

```diff
@@ -48,10 +49,16 @@ export function AsyncMigrations(): JSX.Element {
                                 onClick={() => triggerMigration(asyncMigration.id)}
                             />
                         ) : status === 1 ? (
-                            <StopOutlined
-                                className="migration-btn danger"
-                                onClick={() => forceStopMigration(asyncMigration.id)}
-                            />
+                            <div>
+                                <PlayCircleOutlined
+                                    className="migration-btn success"
+                                    onClick={() => resumeMigration(asyncMigration.id)}
```

Comment:
there's a problem here: people could click this multiple times, could be fixed in a follow-up. Potentially something similar to https://github.com/PostHog/posthog/blob/master/posthog/tasks/async_migrations.py#L60 checking the active task ids in `is_migration_resumable` function.

Reply:
let's not let the migration resume if it is in a running state. probably should go in this pr

Reply:
I started reworking this. I think we should: 
1. by default not roll back. 
2. In the UI have these options: running state -> stop ; errored state -> rollback & resume;